src/modules/curtainBuilder/components/steps/window/containers/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from "react";
+import { useFormikContext, FormikValues } from "formik";
+import WindowStep from "../components";
+import { WindowTypeEnum, WindowQuantityEnum } from "../types";
+
+const WindowStepContainer = () => {
+  const { values, setValues } = useFormikContext<FormikValues>();
+  const [isTypeEmpty, setIsTypeEmpty] = useState(!values.type);
+  const [isDistanceDisabled, setIsDistanceDisabled] = useState(
+    values.quantity === WindowQuantityEnum.single
+  );
+  const [isTripleLeaf, setIsTripleLeaf] = useState(false);
+
+  console.log(values);
+
+  const getImagePath = (): string => {
+    const { singleLeaf, doubleLeaf, tripleLeaf } = WindowTypeEnum;
+    const { single, double } = WindowQuantityEnum;
+
+    switch (values.type) {
+      case singleLeaf:
+        return values.quantity === single
+          ? `${singleLeaf}-${single}.png`
+          : `${singleLeaf}-${double}.png`;
+      case doubleLeaf:
+        return values.quantity === single
+          ? `${doubleLeaf}-${single}.png`
+          : `${doubleLeaf}-${double}.png`;
+      case tripleLeaf:
+        return `${tripleLeaf}-${single}.png`;
+      default:
+        return `${doubleLeaf}-${double}.png`;
+    }
+  };
+
+  const handleSelectOptions = (e: React.MouseEventHandler<HTMLLIElement>) => {
+    const { value } = e.target.dataset;
+    //If no value in select disable all inputs
+    if (!value) {
```

Comment:
This can be simplified by casting value to boolean
`const isEmptyType = !!value; 
setIsTypeEmpty(isEmptyType)
`